libass/ass.c

```diff
@@ -60,8 +72,14 @@ struct parser_priv {
     int check_readorder;
 
     int enable_extensions;
+
+    // track which [Script Info] headers have been set by the script
+    uint32_t sinfo_flags;
```

Comment:
What name would you suggest ? `script_sinfo_flags`, `track_sinfo_flags`, … ?

Reply:
`script_info_flags` or simple `header_flags`.

Reply:
I mainly added the comment to make clear that it SINFO_* only indicates if an Header has been set, **by the script file**/stream itself and not eg changed the regular default value for legacy cases.
This seems more sensible too me as otherwise the track->`header` value can just be used directly.
Expanding `sinfo` or swapping it for `header` doesn't make this clear imho. You can of course argue that these are better variable names in general.

Reply:
I renamed `sinfo_flags` to `header_flags`.